examples/compose.js

```diff
@@ -68,7 +69,7 @@ function compose (...composables) {
     staticPropertyDescriptors: assign({},
       ...getDescriptorProps('staticPropertyDescriptors', composables)),
     initializers: [].concat(...getDescriptorProps('initializers', composables))
-      .filter(initializer => initializer !== undefined),
+      .filter(initializer => isFunction(initializer)),
```

Comment:
Same could be said about my use of `isUndefined` [here](https://github.com/stampit-org/stamp-specification/blob/koresar-patch-1/examples/compose.js#L31).